avm/res/network/network-manager/security-admin-configuration/rule-collection/rule/main.bicep

```
@@ -58,10 +58,10 @@ param priority int
 param protocol string
 
 @sys.description('Optional. List of destination port ranges. This specifies on which ports traffic will be allowed or denied by this rule. Provide an (*) to allow traffic on any port. Port ranges are between 1-65535.')
-param sourcePortRanges array = []
+param sourcePortRanges sourcePortRangesType?
```

Same comment as before
```suggestion
param sourcePortRanges string[]?
```